README.md

```diff
@@ -76,3 +76,8 @@ public OnDialogPerformed(playerid, dialog[], response, success)
     return 1;
 }
 ```
+
+## Credits
+Emmet_ - for easyDialog
+Southclaws - for samp-ctl
```

Comment:
I hate being pedantic [but this is technically sampctl](https://github.com/Southclaws/sampctl/wiki/What-Does-sampctl-Mean)